mfgtool/imx8mm-lpddr4-evk/fullimage.uuu.in

```diff
@@ -0,0 +1,17 @@
+
+uuu_version 1.2.39
+
+SDP: boot -f binaries/imx-boot
+
+SDPV: delay 1000
+SDPV: write -f binaries/imx-boot -skipspl
+SDPV: jump
+
+FB: ucmd setenv fastboot_dev mmc
+FB: ucmd setenv mmcdev ${emmc_dev}
+FB: ucmd mmc dev ${emmc_dev}
+FB: flash -raw2sparse all binaries/@@IMAGE_BASENAME@@-@@MACHINE@@.wic.gz
```

Comment:
This may be a typo: not `wic.gz`, but `.wic` - we may not want to flash a compressed image ;-)